kvmagent/kvmagent/plugins/vms/vm_host_file_monitor.py

```
@@ -0,0 +1,211 @@
+        logger.debug('vm host file monitor started')
+
+    # -- internal --------------------------------------------------------------
+
+    def _get_running_vm_uuids(self):
+        # type: () -> set
+        if self._get_running_vms_func is None:
+            return set()
+        try:
+            vms = self._get_running_vms_func()
+            return set(vm.uuid for vm in vms)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)